packages/core/src/routes/OidcRoutes.js

```diff
@@ -47,7 +44,7 @@ const OidcRoutes = ({
 
   switch (path) {
     case callbackPath:
-      return <CallbackComponent callbackComponentOverride={callbackComponentOverride} />;
```

Comment:
nice update @youf-olivier ,
you have to update redux package too.

Reply:
The problem was in the context package only

Reply:
May be but thos remove break the feature on redux pacakge :p

Reply:
I Don't think so, the redux package didn't use the callbackComponentOverride prop. Or I missed something else ? 

From oidc.js in Redux Package : 

return (
    <OidcProvider store={store} userManager={getUserManager()}>
      <OidcRoutes
        configuration={configuration}
        notAuthenticated={notAuthenticated}
        notAuthorized={notAuthorized}
        callbackComponent={withComponentOverrideProps(
          AuthenticationCallback,
          callbackComponentOverride
        )}
      >
        {children}
      </OidcRoutes>
    </OidcProvider>
  );

Reply:
yep